bench/memory.js

```diff
@@ -1,9 +1,7 @@
-var leveldown = require('../')
-
-var addr = '1111111111111111111114oLvT2'
-
-var db = leveldown(process.env.HOME + '/iterleak.db')
-var records = {
+const leveldown = require('../')
+const addr = '1111111111111111111114oLvT2'
+const db = leveldown(process.env.HOME + '/iterleak.db')
```

Comment:
Shouldn't standard bark at this for not using `path.join()`?

Reply:
It def _should_. Maybe because `__dirname` isn't used?

Reply:
cc @feross

Reply:
Ah, yes. This is not being caught because it doesn't use `__dirname`. Will see if I can tighten this rule up, but there are limits to what static analysis can do  :)